src/main/java/org/broadinstitute/hellbender/engine/ReadWalker.java

```diff
@@ -18,6 +19,9 @@
  *
  * If multiple sources of reads are specified, they are merged together into a single sorted stream of reads.
  *
+ * Reads will be filter with {@link #makeReadFilter()} and transformed with {@link #makePostReadFilterTransformer()} before pass
+ * them to apply().
```

Comment:
This comment needs updating - I think it was from before we had pre and post transformers.